paymentsheet/src/androidTest/java/com/stripe/android/paymentsheet/PaymentSheetBillingConfigurationTest.kt

```diff
@@ -29,13 +29,14 @@ internal class PaymentSheetBillingConfigurationTest {
 
     @Test
     fun testPayloadWithDefaultsAndOverrides() {
-        networkRule.enqueue(
-            method("POST"),
-            path("/v1/consumers/sessions/lookup"),
-        ) { response ->
-            response.setResponseCode(500)
+        repeat(2) {
```

Comment:
These were technically failing before, it just wasn't being caught. https://github.com/stripe/stripe-android/pull/7201/files#diff-00199e88e08d1fa1611f9e90ad0d7f00be39d32388131c2528aba560107d699eR64 fixed that.